test/benchmark/sqlite/summary.json

```diff
@@ -1,14 +1,24 @@
 {
     "benchmarks": [
-        "sqlite/ext2_deletes_between",
-        "sqlite/ext2_deletes_individual",
-        "sqlite/ext2_refill_replace",
-        "sqlite/ext2_selects_ipk",
-        "sqlite/ext2_selects_text_pk",
-        "sqlite/ext2_updates_between",
-        "sqlite/ext2_updates_individual",
-        "sqlite/ext2_updates_big_one",
-        "sqlite/ext2_vacuum",
-        "sqlite/ext2_total"
```

Comment:
@grief8 Since the current code is incorrect, why is the [summary chart rendered correctly](https://asterinas.github.io/benchmark/sqlite/)? And after the fix, will the summary chart be broken?

Reply:
`sqlite/` is just redundant. Benchmark action can handle this situation. I will modify the [README](https://github.com/asterinas/asterinas/blob/main/test/benchmark/README.md#step-3-update-suites-summaryjson) later.

Reply:
#1716 proposes the fix.